Laboratoare/Lab/src/polimorfism/birdcontrollerex1/hierarchyex5/model/RunningShoes.java

```diff
@@ -0,0 +1,32 @@
+package polimorfism.birdcontrollerex1.hierarchyex5.model;
+
+
+public class RunningShoes extends SportShoes {
+
+    int adherence = 150;
+    int amortization = 200;

```

Comment:
la fel ca mai sus, nu initializa aici field-urile ci in constructor.
De asemenea, nu mai lasa field-uri cu default access modifier, cum e aici. Fa-le private! gen private int amortization;